### ISSUE TITLE
Test for WCAG technique H65: Using the title attribute to identify form controls when the label element cannot be used

### ISSUE THREAD

**Issue by kevee** [quailjs/quail#64]
Create a test for the following WCAG 2.0 technique: http://www.w3.org/TR/WCAG20-TECHS/H65.html
